src/Utilities/ProcessorArchitecture.cs

```diff
@@ -16,17 +16,17 @@ namespace Microsoft.Build.Utilities
     static public class ProcessorArchitecture
     {
         // Known processor architectures
-        public const string X86 = "x86";
-        public const string IA64 = "IA64";
+        internal const string X86 = "x86";
```

Comment:
>internal [](start = 8, length = 8)

This broke the compilation of Microsoft.Build.Tasks because it's using them.  Can I include Tasks in InternalsVisibleTo? Or should I leave these as public and provide XML doc comments?